src/main/java/Restriction.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Arrays;
+
+public class Restriction {
+    private static final int CORRECT_INPUT_LENGTH = 3;
+    private static final int CORRECT_INPUT_CHAR_MAX = 57;
+    private static final int CORRECT_INPUT_CHAR_MIN = 49;
+
+    public boolean checkInputIsCorrect(String input) {
+        int inputLength = (int) Arrays.stream(input.split("")).distinct().count();
```

Comment:
```suggestion
        int inputLength = (int) Arrays.stream(input.split(""))
                      .distinct()
                      .count();
```
한 줄에 하나의 ``.``으로 가능하면 코드 포맷을 지켜주시면 좋을 것 같습니다. 
(distinct()와 count()가 코멘트로 코드를 수정하니 코드 포맷(=띄어쓰기)이 원하는 대로 이동이 잘 안되네요.)

Reply:
한줄에 하나의 .으로 코드 포맷을 지킬 수 있도록 유념하겠습니다. 지적 감사합니다.